simuleval/evaluator/instance.py

```diff
@@ -141,6 +176,11 @@ def send_source(self, config_dict: Optional[Dict]):
             )
             self.step += 1
 
+            #For ATDScore
+            self.src_times["NCA"].append(self.step)
```

Comment:
Actually you may just implement the NCA version. We use the this option here to automatically change NCA and CA https://github.com/facebookresearch/SimulEval/blob/main/simuleval/evaluator/scorers/latency_scorer.py#LL35C33-L35C33